editor/components/page-attributes/check.js

```diff
@@ -1,22 +1,10 @@
 /**
  * External dependencies
  */
-import { connect } from 'react-redux';
 import { get } from 'lodash';
 
-/**
- * WordPress dependencies
- */
-import { withAPIData } from '@wordpress/components';
-import { compose } from '@wordpress/element';
-
-/**
- * Internal dependencies
- */
-import { getCurrentPostType } from '../../store/selectors';
-
-export function PageAttributesCheck( { postType, children } ) {
-	const supportsPageAttributes = get( postType.data, [
+export default function PageAttributesCheck( { postType, children } ) {
```

Comment:
I built a generic component to achieve this. You can reuse it here `PostTypeSupportSupportCheck`.

But I'm wondering why did you remove the "connect" from here.